### DISCUSSION TITLE
Red dead redemption 2 from epic

### DISCUSSION THREAD

**Discussion by michalchamow** [ebenbruyns/junkstore#55] [General]
Hi, need help with my red dead redeption 2 installation. It installed all correctly, game is visible in steam, I signed in to rockstar launcher but I was not able to run the game. It shows Rockstar error code 134 and closes. I have stable internet connection. Any ideas how to solve this?

**Comment by mrsjunkrunner** [comment]
hey have you checked out our wiki? [Red Dead Redemption 2](https://wiki.junkstore.xyz/wiki/Help:Epic_Games/Working/Read_Dead_Redemption_2)

**Comment by mrsjunkrunner** [comment]
> Hi, need help with my red dead redeption 2 installation. It installed all correctly, game is visible in steam, I signed in to rockstar launcher but I was not able to run the game. It shows Rockstar error code 134 and closes. I have stable internet connection. Any ideas how to solve this?

You manage to get this working? If not a good place to ask is in our discord server. We don't actually have this game, but a lot of people in our server do. That is where we got the work around from that I put on our wiki.

**Comment by Amerc0** [comment]
The problem still exists and Rockstar Games causes a lot of trouble...

**Reply by mrsjunkrunner** [reply]
Hey this was closed last year becuase at the time the fix I linked to on our wiki was the method to get RDR2 and GTA V working. However, in the past 11 months there have been various updates which have made Rockstar games more difficult to get working.

Unfortunately, we do not have any Rockstar games purchased via Epic, so we would need support and input from the community to find a solution to get these games working consistently via Junk Store.

As there currently is no integrated solution for Rockstar (like Epic, Gog and Amazon games) this will require a large amount of effort to get working correctly. We welcome contributions and PR's from the community, as we only have one dev and myself (non-dev) working on and supporting Junk Store at the moment.

**Comment by Akio333** [comment]
I got RDR2 working via Junk Store.
I did below steps:
- Use Proton 11
- Followed steps for windows given on [Heroic Docs](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Rockstar-Games-from-Epic-Games)
- Updated target to `fix.bat` file
- If error 18 coming for social club then edit settings.xml and use DX12 instead of vulkan.
